ext/dapr-ext-workflow/dapr/ext/workflow/workflow_runtime.py

```diff
@@ -28,8 +29,9 @@ class WorkflowRuntime:
     """WorkflowRuntime is the entry point for registering workflows and activities.
     """
 
-    def __init__(self):
-        self.__worker = worker.TaskHubGrpcWorker()
+    def __init__(self, host: Union[str, None] = None, port: Union[str, None] = None):
```

Comment:
nit: should we use `Option` instead of `Union` here?
```suggestion
    def __init__(self, host: Option[str] = None, port: Option[str] = None):
```

Reply:
`Option` is not a thing anyway, it's called `Optional` and you must import `from typing import Optional`

Reply:
Yes please use Optional instead of Union[something, None]